README.md

```diff
@@ -103,16 +103,52 @@ cd ./qs
 start-re-manager --config=./qs-config.yml
 ```
 
+### Warnings (queueserver)
+
+The QS host process writes files into the `qs/` directory. This directory can be
+relocated. However, it should not be moved into the instrument package since
+that might be installed into a read-only directory.
+
```

Comment:
I had moved this to the bottom so that we can incorporate multiple warnings. Not exclusively qs ones.

Reply:
Seemed lonely without others.  Moving back to the bottom.